### PR TITLE
Add Your Matt Holovach

### PR DESCRIPTION
**Pull request by seoMattH**
Added this third one down

{
    name: 'Matt Holovach',
    description: 'Loves SEO, improving coding skills and good food.',
    url: 'https://www.coloradoseodesign.com/',
    twitter: '@HolovachMatt'
    emoji: '🚀',
    country: 'USA',
    computer: 'microsoft',
    phone: 'android',
    tags: ['Developer', 'Alpine.js', 'PHP', 'JavaScript', 'CSS', 'Gulp', 'Perch CMS', 'HTML', 'Node.js'],
  }

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -28,6 +28,16 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Matt Holovach',
+    description: 'Loves SEO, improving coding skills and good food.',
+    url: 'https://www.coloradoseodesign.com/',
```
Comment (jorgeruvalcaba) [inline src/data.js:34]: This should direct to a uses page

File: src/data.js [hunk 2]
```diff
@@ -28,6 +28,16 @@ module.exports = [
...
+    emoji: '🚀',
+    country: 'USA',
+    computer: 'microsoft',
```
Comment (jorgeruvalcaba) [inline src/data.js:37]: this should be **microsoft**

File: src/data.js [hunk 3]
```diff
@@ -28,6 +28,16 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Matt Holovach',
+    description: 'Loves SEO, improving coding skills and good food.',
+    url: 'https://www.coloradoseodesign.com/',
+    emoji: '🚀',
+    country: 'USA',
```
Comment (jorgeruvalcaba) [inline src/data.js:36]: You should put usa emoji here

File: src/data.js [hunk 4]
```diff
@@ -28,6 +28,16 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Matt Holovach',
+    description: 'Loves SEO, improving coding skills and good food.',
```
Comment (jorgeruvalcaba) [inline src/data.js:33]: (optional) could you remove the dot at the end of the line?

**Review by jorgeruvalcaba — COMMENTED** [review]
Left some comments @seoMattH

**Review by davidak — CHANGES_REQUESTED** [review]
see comments

File: src/data.js [hunk 5]
```diff
@@ -28,6 +28,16 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Matt Holovach',
+    description: 'Loves SEO, improving coding skills and good food',
+    url: 'https://www.coloradoseodesign.com/uses.php',
+    emoji: '✝',
+    country: 'us',
```
Comment (davidak) [inline src/data.js:36]:
```suggestion
    country: '🇺🇸',
```
this is the correct emoji

File: src/data.js [hunk 6]
```diff
@@ -28,6 +28,16 @@ module.exports = [
...
+    computer: 'microsoft',
```
Comment (davidak) [inline src/data.js:37]: @jorgeruvalcaba you mean

```suggestion
    computer: 'windows',
```

? it is `microsoft`, but that is wrong :D

**Review by davidak — CHANGES_REQUESTED** [review]
@seoMattH thanks for updating the PR.

Can you accept my suggestions?

**Review by davidak — APPROVED** [review]
Great, thank you!

**Pull request merged**